katana/static/app/controllers/testcaseCapCtrl.js

```diff
@@ -1299,7 +1447,7 @@ $scope.showRules = function(execType){
                 else{
                     if($scope.status.nodatafile == '0'){
                     swal({
-                        title: "Kindly provide the correct Relative path for Input data File, if auto-population of system & Subsystem name is needed.",
+                        title: "Kindly provide the correct Relative path for Input data File and Jocket File, if auto-population of system & Subsystem name is needed.",
```

Comment:
Use Wrapper file

Reply:
-Made necessary changes according to your suggestions and committed.